spring-batch-core/src/main/java/org/springframework/batch/core/step/item/FaultTolerantChunkProcessor.java

```diff
@@ -149,6 +149,12 @@ protected void initializeUserData(Chunk<I> inputs) {
 			inputs.setUserData(data);
 			data.setOutputs(new Chunk<O>());
 		}
+		else {
```

Comment:
Along the same lines as the license comment above, when we work on files with a license in the header, please be sure to update the date in the license to the current year.  Also, don't be afraid to add your author tag as well.

Reply:
Sure! I will do it next time. Thanks